mqtt.kafka.reflect/chart/files/zilla.yaml

```diff
@@ -39,11 +39,31 @@ bindings:
     routes:
       - when:
           - topic: zilla
-        exit: fan_server0
-  fan_server0:
-    type: fan
-    kind: server
-    exit: echo_server0
-  echo_server0:
-    type: echo
-    kind: server
+          - topic: zilla2
```

Comment:
For the previous `mqtt.reflect` example using `fan` plus `echo`, we could not differentiate between topic names, so we had to restrict it to just one topic here.

Now with `mqtt.kafka.reflect` example, we are no longer constrained by that, so rather than explicitly listing topics here, suggest you remove the constraint and `exit` unconditionally.